utils/llvm-symbolizer-client.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+import urllib.request
+import sys
+
+
+def run():
+    url = "http://localhost:43210"
```

Comment:
I guess this port number is arbitrary (which is fine).
I also guess that we can guarantee that no other process uses this port number because the symbolizer_server.py is started at the beginning of any CI job that uses sanitizers. So symbolizer_server.py always starts before any (random) port numbers are assigned to started arangod instances.

Reply:
Yes, it is arbitrary, and yes, it is always started before anything else. :)